### ISSUE TITLE
Compile time i18n

### ISSUE THREAD

**Issue by botika** [botika/yarte#192]
Just use this simple concepts https://github.com/terry90/internationalization-rs

**Comment by raymclee** [comment]
does the t! marco able to use inside the template?

**Comment by botika** [comment]
> does the t! marco able to use inside the template?

Yes. But we are refactoring the parser and yarte to use it from wasm and improve compile times as well as making a dev center with live reload etc. But I'm sleeping in a Spanish jail because the idiots of the Spanish Justice say that I'm a criminal. And of course I don't have much time.
